mmv1/products/compute/RegionResizeRequest.yaml

```diff
@@ -0,0 +1,413 @@
+    base_url: '{{op_id}}'
+    kind: 'compute#operation'
+    path: 'name'
+    wait_ms: 1000
+  result:
+    path: 'targetLink'
+    resource_inside_response: false
+  error:
+    path: 'error/errors'
+    message: 'message'
+### Update method ###
+# Resize requests are currently not update-able
+
+### Delete Method ###
+# Custom delete method to handle resize request cancellations vs. deletions.
+# If a resize request is in the CREATING or ACCEPTED state, it must be canceled before it can be
+# deleted. If a resize request is NOT in any of the mentioned state, it can be directly deleted.
+custom_code:
+  custom_delete: 'templates/terraform/custom_delete/compute_rmig_resize_request_delete.go.tmpl'
+# Examples for testing
+examples:
+- name: 'compute_rmig_resize_request'
+  min_version: beta
+  primary_resource_id: 'a3_resize_request'
+  vars:
+    resize_request_name: 'a3-dws'
+# Resize request parameters injected via URL
+parameters:
+- name: 'region'
+  type: ResourceRef
+  description: |
+    Name of the compute region scoping this request. Name should conform to RFC1035.
+  url_param_only: true
+  required: true
+  resource: 'Region'
+  imports: 'name'
+- name: 'instanceGroupManager'
+  type: ResourceRef
+  description: |
+    The name of the regional managed instance group. The name should conform to RFC1035 or be a resource ID.
+    Authorization requires the following IAM permission on the specified resource instanceGroupManager:
+    *compute.instanceGroupManagers.update
+  url_param_only: true
+  required: true
+  resource: 'InstanceGroupManager'
+  imports: 'name'
+# Non-URL parameters including input and output parameters
+properties:
+- name: 'creationTimestamp'
+  type: Time
+  description: |
+    The creation timestamp for this resize request in RFC3339 text format.
+  output: true
+- name: 'state'
+  type: Enum
+  description: |
+    [Output only] Current state of the request.
+  output: true
+  enum_values:
+  - 'CREATING'
+  - 'ACCEPTED'
+  - 'FAILED'
+  - 'SUCCEEDED'
+  - 'CANCELLED'
+- name: 'name'
+  type: String
+  description: |
+    The name of this resize request. The name must be 1-63 characters long, and comply with RFC1035.
+  required: true
+- name: 'description'
+  type: String
+  description: |
+    An optional description of this resize-request.
+- name: 'resizeBy'
+  type: Integer
+  description: |
+    The number of instances to be created by this resize request. The group's target size will be increased by this number.
+  required: true
+- name: 'requestedRunDuration'
+  type: NestedObject
+  description: |
+    Requested run duration for instances that will be created by this request. At the end of the run duration instances will be deleted.
+  properties:
+  - name: 'seconds'
+    type: String
+    description: |
+      Span of time at a resolution of a second. Must be from 0 to 315,576,000,000 inclusive. Note: these bounds are computed from: 60 sec/min * 60 min/hr * 24 hr/day * 365.25 days/year * 10000 years
```

Comment:
do we really allow instances to run for 10000 years ?

Reply:
No. Obviously not. Minimum 10 minutes and maximum 7days. 
IIUC, this description was copied from mixer resource where duration is a nested resource of other resource. Since the duration was a common nested resource for any other resource (not only dws), it has a generic description for seconds.
I modified the description, since the resource is now specific to dws. Thanks!